README.md

```diff
@@ -1,12 +1,10 @@
 # OSFR Launcher
-A launcher for the Open-Source FreeRealms Project:
-
-https://github.com/Open-Source-Free-Realms/OpenSourceFreeRealms
+A launcher for the [Open-Source FreeRealms Project](https://github.com/Open-Source-Free-Realms/OpenSourceFreeRealms)
 ![Alt text](https://github.com/Lillious/OSFR-Launcher/blob/main/teaser.png?raw=true)
 
 # Install Instructions
 
-- Download **launcher-win32-x64.zip** from the latest release located at: https://github.com/Lillious/OSFR-Launcher/releases/
+- Download **launcher-win32-x64.zip** from the [latest release](https://github.com/Lillious/OSFR-Launcher/releases/)

```

Comment:
```suggestion
- Download **launcher-win32-x64.zip** from the [latest release](https://github.com/Open-Source-Free-Realms/OSFR-Launcher/releases/)
```
Shouldnt it be like this?

Reply:
nice catch, guess I missed updating that link